roles/redfish_storage_volume/molecule/RAID0/converge.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Converge
+  hosts: all
+  vars:
+    redfish_storage_volume_failure: {}
+    redfish_storage_volume_raid: "RAID0"
+    redfish_storage_volume_minimum_drives: 2
```

Comment:
raid 0 minimum is 2 or 1?

Reply:
![image](https://github.com/dell/dellemc-openmanage-ansible-modules/assets/39791839/f05abe28-9eaf-445f-89e0-97f900b1127a)